.github/workflows/tests.yml

```diff
@@ -71,6 +72,7 @@ jobs:
       matrix:
         python-version:
           - "3.12"
+          - "3.13"
```

Comment:
Ideally we probably only want 1 version during normal PRs to limit the amount of time it takes for checks to run, so either 3.12 or 3.13 here.